src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
```

Comment:
Luois always suggests to use React.useState rather than this.

Reply:
you already commented this on Jennet’s PR and Louis said that you should use whatever is consistent with code base. Jennet and Jundi use useState, so I used it too. probably you should use it too for consistency.

Reply:
By the way, who is Luois ? I only know Louis 😂😂